modules/ebi-metagenomics/generategaf/meta.yml

```diff
@@ -0,0 +1,46 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "generategaf"
+description: Script that generates a GO Annotation File (GAF) out of an InterProScan result tsv file.
```

Comment:
GAF is official name? should it be GOAF? (I think inclusion of GO brings more understanding)

Reply:
Yeah GAF is the official name: https://geneontology.org/docs/go-annotation-file-gaf-format-2.1/